temporalcli/commandsgen/commands.yml

```diff
@@ -3256,6 +3224,7 @@ option-sets:
           - UseExisting
           - TerminateExisting
   - name: payload-input
+    description: Commands accepting an input payload
```

Comment:
Leaving any personal preferences aside, "Commands that accept an input payload" is more aligned with all previous descriptions

Reply:
removing `description` [per comment above](https://github.com/temporalio/cli/pull/693/files#r1803621328)